.github/workflows/build-binaries.yml

```diff
@@ -104,6 +127,18 @@ jobs:
             *.tar.gz
             *.sha256
 
+      # uv-build
+      - name: "Build wheels uv-build - x86_64"
+        uses: PyO3/maturin-action@v1
+        with:
+          target: x86_64
+          args: --profile minimal-size --locked --out crates/uv-build/dist -m crates/uv-build/Cargo.toml
+      - name: "Upload wheels uv-build"
+        uses: actions/upload-artifact@v4
+        with:
+          name: wheels_uv_build-macos-x86_64
+          path: crates/uv-build/dist
+
```

Comment:
Hmm I was gonna say this will slow down releases a lot if it's serial but in theory this thing should build super fast so it's fine?

Reply:
It's 22s on a fresh target directory for me locally. I had initially put them in succession to reuse the target dir, but that mostly doesn't apply anymore with the different opt mode.